debian/rules

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/make -f
+
+BUILDDIR = build_dir_debian
+
+%:
+	dh $@ --parallel
+
+override_dh_auto_configure:
+	mkdir $(BUILDDIR)
+	cd $(BUILDDIR) && cmake .. -DCMAKE_BUILD_TYPE=Release -DWITH_QT4=1 -DWITH_CLIENT=1 -DWITH_SERVER=1 -DWITH_ORACLE=1 -DCMAKE_INSTALL_PREFIX:PATH=/usr
```

Comment:
By multi-build I mean latest ubuntu desktop + latest ubuntu LTS. If it's in any way a burden I'm fine saying we only support latest version period but I don't think it'll actually be that hard.

The split build is also something that we probably don't need to do